rules/mocha.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  plugins: [
+    'mocha',
+  ],
+  rules: {
+    'mocha/no-exclusive-tests': 'error',
+    'mocha/no-skipped-tests': 'off',
+    'mocha/no-pending-tests': 'error',
+    'mocha/handle-done-callback': 'error',
+    'mocha/no-synchronous-tests': 'off',
+    'mocha/no-global-tests': 'error',
+    'mocha/no-return-and-callback': 'error',
+    'mocha/valid-test-description': 'off',
+    'mocha/valid-suite-description': 'off',
+    'mocha/no-sibling-hooks': 'off',
```

Comment:
I don't see a use case though. This is something I did a few times already but it was by accident every time.